packages/data-visualization/src/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Stat';
+export * from './StatProgress';
```

Comment:
it looks like `tsc` got run and it generated all of these `.d.ts` types, could you please remove them since they won't be needed?